dotcom-rendering/src/components/SvgGuardianAustraliaLogo.tsx

```diff
@@ -8,51 +13,51 @@ export const SvgGuardianAustraliaLogo = () => {
-					fill="#FFE500"
+					fill={yellow}
 				/>
 				<path
 					d="M76.9883 58.5507L82.8105 55.5129V9.6563H78.4077L67.6534 23.9533H66.4385L67.1242 8.01685H113.75L114.424 23.9533H113.149L102.611 9.6563H98.0998V55.4165L103.958 58.5146V60.0696H76.9883V58.5507Z"
-					fill="white"
+					fill={white}
```

Comment:
This could be applied to the parent and omitted for all children except when different. The fill property cascades.